scripts/main.js

```diff
@@ -31,6 +32,65 @@ module.exports = (robot) => {
             }
         }
     })
+    
+    robot.hear(/bot create milestone|bot close milestone/i, (res) => {
+        var message = res.message.text
+        message = message.split(' ')
+        if(message.length === 7 && message[0] === "bot" && message[1] === "create" && message[2] === "milestone") {
+          var repos = message[6]
+          repos = repos.split('/')
+          var title = message[3]+" "+message[4]+" "+message[5]
```

Comment:
let.
Again, Title should be anything - not just `message[3]+" "+message[4]+" "+message[5]`

Reply:
@zelima Let's keep it in this way, for `icebox` and `backlog` I will find out something else